mailpoet/tests/integration/REST/Automation/Workflows/WorkflowsCreateFromTemplateTest.php

```diff
@@ -35,4 +60,21 @@ public function testWorkflowsCreatedFromTemplatesAreCreatedInDraftStatus(): void
     expect($createdWorkflow->getStatus())->equals('draft');
   }
 
+  public function testWorkflowsCreatedFromTemplatesReturnsWorkflowId(): void {
+    $storage = ContainerWrapper::getInstance()->get(WorkflowStorage::class);
```

Comment:
I guess you can use `$this->workflowStorage` here.

Reply:
Addressed in 0038088ee